ml-proto/spec/kernel.ml

```diff
@@ -119,16 +119,18 @@ and func' =
 type memory = memory' Source.phrase
 and memory' =
 {
-  initial : Memory.size;
+  min : Memory.size;
```

Comment:
I see, good point. Well, I'll change the name to whatever AstSemantics and BinaryEncoding will eventually agree on. :)